packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react';
+
+import ExpoScreenCapture from './ExpoScreenCapture';
+
+/**
+ * Prevents screenshots and screen recordings. If you are
+ * already preventing screen capture, this method does nothing.
+ *
+ * On iOS, this will only prevent screen recordings, and is only
+ * available on iOS 11 and newer. On older iOS versions, this method
+ * does nothing.
+ *
+ * @example
+ * ```typescript
+ * preventScreenCapture();
+ * ```
+ */
+export async function preventScreenCapture() {
+  return ExpoScreenCapture.preventScreenCapture();
+}
```

Comment:
Three things:

1. If the platform layer (specifically on Android) throws an error, it will reject the promise returned from `ExpoScreenCapture.preventScreenCapture()`. So if developers want to catch this error, they need to await `preventScreenCapture()` (or add a rejection handler to the promise they get).
2. Add an explicit return type.
3. Use `await` instead of `return` here since we don't want to return the `null` we get from the platform layer call.

```suggestion
export async function preventScreenCaptureAsync(): Promise<void> {
  await ExpoScreenCapture.preventScreenCapture();
}
```